### PR TITLE
4.0

### PR DESCRIPTION
**Pull request by PikaDude**
Version 4.0 of the extension.

- Added Wild World/City Folk/Let's Go to the City Snowing and Raining weather alternatives
- Made time checker more accurate
- Made Town Tune Editor look nicer
- Added Live Weather option
- Decreased volume step amount
- Add option to toggle badge text
- Use Chrome Bootstrap on the options page
- Made weather and game options separate
- Added Animal Crossing Snowing weather alternative
- Updated extension icon
- Added more extension badges
- Updated K.K. Slider icon to a New Leaf render
- Removed white background on hourly notification icon
- Added K.K. Slider Aircheck and Live version choices
- Updated badge text to show full hour (E.g. 8pm rather than 8p)
- Added support for Chrome's Media Session service
- Added support for remote streaming for music rather than bundling music with the extension
- Updated every song to be of the highest possible quality, removing any static where possible, bad loops and lowering any excessively high bitrates
- Added every single K.K. Slider song (all 91 songs)
- Reorganised extension's file structure
- Added dynamic Contributors/Credits list
- Added credit links to original authors
- Added support for more desktop notifications
- Added dynamic loop times function
- Added an option to pause or reduce music volume while a tab is playing audio
- Added an option to play the town tune regardless if the extension's music is paused
- Added better error handling
- Polished options page
- Many other various additions, bug fixes, code refactors and changes to make Animal Crossing Music Extension the BEST Animal Crossing extension ever!

### CODE REVIEW

**Review by jordigarcl — APPROVED** [review]
The forked repo is in a great state, and no major issues have been found when integrating the changes here ✌️

**Comment by jordigarcl** [comment]
Giving visibility to the fact that there still are a couple of [PRs opened in the fork](https://github.com/PikaDude/ac-music-extension-revived/pulls), it's probably best to wait until those are merged there as well.

Edit: PRs have been merged here. Lane is clear!

**Comment by PikaDude** [comment]
The PRs opened in the fork have now been merged. Should be good to merge now.

**Pull request merged**
